app/assets/stylesheets/application.css

```diff
@@ -1,339 +0,0 @@
-.user_container {
-  margin: 2em 25em;
-}
-
-.user_title_txt {
-  text-align: center;
-  margin-top: 20px;
-  font-size: 30px;
-}
-
-.user_field {
-  display: flex;
-  align-items: center;
-  justify-content: space-around;
-}
```

Comment:
- [ ] When I run your app locally using the `rails server` command, I get an issue because you deleted a file called "application.css" that is still required in the asset pipeline.

<img width="553" alt="image" src="https://user-images.githubusercontent.com/77190203/173160903-979da061-37c0-4241-ac21-6db6c9f081f5.png">

Reply:
I used bootstrap for the blog app and it worked for me @medaminedev66

Reply:
You can add an `application.css` and `application.js` for it to run the app